Skills/Perk/PregnancyBetterOvulationV2.gd

```diff
@@ -0,0 +1,28 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyBetterOvulationV2
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Breeding overdrive"
+
+func getVisibleDescription():
+	return "Your reproductive organs are pushing themselves to the limit for you to bear as many children as possible. \n" \
+			+ "Maximum of the egg cells you release during each ovulation phase is increased even further and fertility is improved by 20%"
+	
```

Comment:
450% max eggs total with the previous perk.

Reply:
450 sounds like a lot. Kids don't get processed but they do still take up some memory each. Not that much but still. Maybe lower it to 200 with both perks?..

Reply:
Lowered to 250 with both perks (100 base + 50 first + 100 second) for now.